phpseclib/Crypt/TripleDES.php

```diff
@@ -258,6 +258,7 @@ public function setKey($key)
         switch (strlen($key)) {
             case 16:
                 $key.= substr($key, 0, 8);
+                break;
             case 24:
                 break;
             default:
```

Comment:
Thanks for the comment. I suspected, but was not sure if it's a correct behavior